src/components/atom/ImageUploadBasic.tsx

```
@@ -0,0 +1,58 @@
+
+const ImageUploadBasic = (props: IImageUploadBasicProps): ReactElement => {
+    const { onChange, disabled, file } = props
+
+    const { darkMode } = useCoreStore()
+
+    const ref = useRef<HTMLInputElement>(null)
+
+    const [image, setImage] = useState<string>("")
+
+    const handleFileChange = async (e: ChangeEvent<HTMLInputElement>): Promise<void> => {
+        const selectedFile = e.target.files && e.target.files[0]
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)